requirements.in

```diff
@@ -0,0 +1,17 @@
+ArchivesSnake==0.9.1
+asterism==0.7.2
+Django==3.2.5
+django-cron==0.5.1
+djangorestframework==3.12.4
+ElectronBonder==0.7.2
+health-check==3.4.1
+iso-639==0.4.5
+jsonschema==3.2.0
+odin==1.7.0
+psycopg2-binary==2.9.1
+PyYAML==5.4.1
+rac-schemas==0.27
+requests==2.25.1
```

Comment:
I think `requests` isn't top-level.

Reply:
https://github.com/RockefellerArchiveCenter/pisces/blob/base/fetcher/helpers.py#L1

Reply:
I removed from top-level in [cartographer_backend](https://github.com/RockefellerArchiveCenter/cartographer_backend/pull/51#discussion_r664904745). Is there a different implementation context that makes it different between these repos?